ccloud/docs/beginner-cloud.rst

```diff
@@ -637,73 +634,38 @@ Run kafka-connect-datagen connector with wildcard ACLs
 
-   .. code-block:: text
-
-         DATA=$( cat << EOF
-         {
-            "name": "datagen-demo-topic-3",
-            "config": {
-              "connector.class": "io.confluent.kafka.connect.datagen.DatagenConnector",
-              "kafka.topic": "demo-topic-3",
-              "quickstart": "pageviews",
-              "key.converter": "org.apache.kafka.connect.storage.StringConverter",
-              "value.converter": "org.apache.kafka.connect.json.JsonConverter",
-              "value.converter.schemas.enable": "false",
-              "max.interval": 5000,
-              "iterations": 1000,
-              "tasks.max": "1"
-            }
-         }
-         EOF
-         )
-
-         curl --silent --output /dev/null -X POST -H "Content-Type: application/json" --data "${DATA}" http://localhost:8083/connectors
-
-
-#. Wait about 20 seconds for the kafka-connect-datagen connector to start producing messages.
+      2020-12-16T16:10:09.732-0700 [DEBUG] ConnectService.Create request: POST https://confluent.cloud/api/accounts/env-5qz2q/clusters/lkc-x6m01/connectors Body:{"name":"datagen_ccloud_pageviews","config":{"connector.class":"DatagenSource","iterations":"1000000000","kafka.api.key":"ESN5FSNDHOFFSUEV","kafka.api.secret":"nzBEyC1k7zfLvVON3vhBMQrNRjJR7pdMc2WLVyyPscBhYHkMwP6VpPVDTqhctamB","kafka.topic":"demo-topic-3","max.interval":"500","name":"datagen_ccloud_pageviews","output.data.format":"JSON","quickstart":"PAGEVIEWS","tasks.max":"1"}}
+      2020-12-16T16:10:21.658-0700 [DEBUG] ConnectService.Create response: 201 Created Body: {"name":"datagen_ccloud_pageviews","type":"source","config":{"cloud.environment":"prod","cloud.provider":"aws","connector.class":"DatagenSource","iterations":"1000000000","kafka.api.key":"****************","kafka.api.secret":"****************","kafka.dedicated":"false","kafka.endpoint":"SASL_SSL://pkc-4kgmg.us-west-2.aws.confluent.cloud:9092","kafka.region":"us-west-2","kafka.topic":"demo-topic-3","kafka.user.id":"152738","max.interval":"500","name":"datagen_ccloud_pageviews","output.data.format":"JSON","quickstart":"PAGEVIEWS","tasks.max":"1"},"tasks":[]} request: POST https://confluent.cloud/api/accounts/env-5qz2q/clusters/lkc-x6m01/connectors Body:{"name":"datagen_ccloud_pageviews","config":{"connector.class":"DatagenSource","iterations":"1000000000","kafka.api.key":"ESN5FSNDHOFFSUEV","kafka.api.secret":"nzBEyC1k7zfLvVON3vhBMQrNRjJR7pdMc2WLVyyPscBhYHkMwP6VpPVDTqhctamB","kafka.topic":"demo-topic-3","max.interval":"500","name":"datagen_ccloud_pageviews","output.data.format":"JSON","quickstart":"PAGEVIEWS","tasks.max":"1"}}
+      Created connector datagen_ccloud_pageviews lcc-zno83
 
 #. Run the following command to verify connector is running:
 
    .. code-block:: bash
 
-      curl --silent http://localhost:8083/connectors/datagen-demo-topic-3/status | jq -r '.'
+      ccloud::wait_for_connector_up connectors/ccloud-datagen-pageviews.json 240 || exit 1
```

Comment:
I'm not sure we'd want to have the `exit` conditional in the directions. If we do the user will lose their shell instance if the command doesn't succeed

Reply:
I'm glad you pointed that out. I removed that bit in the instructions and corrected the path to the .json config in this [bc73545](https://github.com/confluentinc/examples/pull/887/commits/bc73545dabaa3e4b8b011366fd24b25b0ee43d22)